### PR TITLE
Fix #25133

### PR DESCRIPTION
**Pull request by yp05327**
Fix #25133

Thanks @wxiaoguang @silverwind.
I'm sorry I made a mistake, it will be fixed in this PR.

### CODE REVIEW

**Review by wxiaoguang — APPROVED** [review]
Never mind, thank you very much!

And .... a small question, what does `data-review-id="dismiss-review-{{.Review.ID}}"` do here?

**Review by lunny — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
> Never mind, thank you very much!
> 
> And .... a small question, what does `data-review-id="dismiss-review-{{.Review.ID}}"` do here?

Seems like a unnecessary class to me, could not find anything when searching `data-review-id` in the codebase except this place.

File: templates/repo/issue/view_content/sidebar.tmpl [hunk 1]
```diff
@@ -58,10 +58,10 @@
 						</div>
 						<div class="gt-df gt-ac gt-gap-3">
 							{{if (and $.Permission.IsAdmin (or (eq .Review.Type 1) (eq .Review.Type 3)) (not $.Issue.IsClosed))}}
-								<a href="#" class="ui muted icon gt-df gt-ac show-modal" data-review-id="dismiss-review-{{.Review.ID}}" data-tooltip-content="{{$.locale.Tr "repo.issues.dismiss_review"}}" data-modal="#dismiss-review-modal">
+								<a href="#" class="ui muted icon gt-df gt-ac show-modal" data-review-id="dismiss-review-{{.Review.ID}}" data-tooltip-content="{{$.locale.Tr "repo.issues.dismiss_review"}}" data-modal="#dismiss-review-modal-{{.Review.ID}}">
```
Comment (silverwind) [inline templates/repo/issue/view_content/sidebar.tmpl]:
```suggestion
								<a href="#" class="ui muted icon gt-df gt-ac show-modal" data-tooltip-content="{{$.locale.Tr "repo.issues.dismiss_review"}}" data-modal="#dismiss-review-modal-{{.Review.ID}}">
```

**Comment by silverwind** [comment]
https://github.com/go-gitea/gitea/pull/25172 contains this backport.

**Comment by yp05327** [comment]
@yardenshoham 
Do we need to remove `backport/manual` label when the manual backport is done?

**Comment by lunny** [comment]
> @yardenshoham Do we need to remove `backport/manual` label when the manual backport is done?

I don't think so.

**Comment by yardenshoham** [comment]
No. The bot ignores all PRs with https://github.com/go-gitea/gitea/labels/backport%2Fdone or https://github.com/go-gitea/gitea/labels/backport%2Fmanual

**Pull request merged**